plugins/aws/ec2/appTierInstanceIamRole.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
             default: ''
         },
     },
+    realtime_triggers: ['ec2:RunInstance'],
```

Comment:
@AkhtarAmir Please note that this is ec2:RunInstances and not ec2:RunInstance. Let's change this and all other occurencs before merging